app/views/hyrax/collections/_show_descriptions.html.erb

```diff
@@ -1,7 +1,8 @@
-<h2 class="sr-only">Descriptions</h2>
-<dl class="dl-horizontal metadata-collections">
```

Comment:
I believe the UX group specifically asked for definition lists here as a usability concern.  @samvera/hyrax-ui-ux-advisors is that right?

Reply:
I would be interested in learning more about this concern.

Reply:
Yes testing with users who use assistive technologies found the key value pairs of `<dl>` more suitable than tables.

Reply:
✅